### ISSUE TITLE
Question: Connect multiple Discord rooms to the same Matrix room

### ISSUE THREAD

**Issue by mentalisit** [Half-Shot/matrix-appservice-discord#674]
Will it be possible to connect multiple Discord rooms to the same matrix room so everyone can chat in the same chat?

**Comment by jonaharagon** [comment]
Wouldn't that send replies to every Discord room too? Discord rooms which wouldn't necessarily have the original message they're replying to. Sounds like it would get confusing.

**Comment by mentalisit** [comment]
Yes, that's exactly what I need.  the goal is to connect 5+ channels of discord with telegrams

**Comment by jaller94** [comment]
Running one bridge per channel is possible and may give you the experience you're looking for.

**Comment by mentalisit** [comment]
> Running one bridge per channel is possible and may give you the experience you're looking for.

on one bridge does not suit me.  I need a lot of bridges in the matrix and the connection between them
